### ISSUE TITLE
Accessibility: ensure that view changes at payoutPreference are noticed to SR

### ISSUE THREAD

**Issue by IagoSRL** [loconomics/loconomics#735]
### Long Description
The implementation of /payoutPreference activity was done by splitting complexity through kocomponents, being the container for all of them `payout-preference-view`. The several components in use are created/displayed depending on triggered actions; this is different to the approach used in other activities, where we navigate between activities implementing specific steps, and at that case, the general logic to focus the activity heading make it works for SR (screen readers) correctly. We need to ensure that happens with the components approach used here.

### Tasks
- Try accessibility of the activity; I think will fail but just in case
- Recently a new Knockout binding was added, `autofocus`, to help assist with this same problem at the Lead Generation components #675, check if that works here
- If previously fails, look for alternative solutions: an idea is to create a component to manage the visualization of a specific view from the set of children passed in (with an attribute to name the view), that will be in charge of managing visualization/creation and focusing the content; a bit like managing the different tab-content elements in a tabbed interface.
